alacritty_terminal/src/term/mod.rs

```diff
@@ -1160,7 +1160,7 @@ impl<T> Term<T> {
         self.set_scrolling_region(1, self.grid.num_lines().0);
 
         // Clear grid
-        let template = Cell { bg: self.cursor.template.bg, ..Cell::default() };
+        let template = self.cursor.template;
```

Comment:
You inlined it every else, might as well do the same here, right? Or does this exceed the 100 column limit?

Reply:
Borrow checker.